SU2_CFD/include/drivers/CDriver.hpp

```diff
@@ -509,17 +519,289 @@ class CDriver : public CDriverBase {
    */
   void SetInletAngle(unsigned short iMarker, passivedouble alpha);
 
+  /*!
+   * \brief Get the heat fluxes at a mesh vertex.
+   * \param[in] iPoint - Point index.
+   * \return Vertex heat flux.
+   */
+  vector<passivedouble> GetHeatFluxes(unsigned long iPoint) const;
+
+  /*!
+   * \brief Get the heat fluxes at a marker vertex.
+   * \param[in] iMarker - Marker index.
+   * \param[in] iVertex - Marker vertex index.
+   * \return Vertex heat fluxes (nVertex, nDim).
+   */
+  vector<passivedouble> GetMarkerHeatFluxes(unsigned short iMarker, unsigned long iVertex) const;
+
+  /*!
+   * \brief Get the normal heat fluxes at a marker vertex.
+   * \param[in] iMarker - Marker index.
+   * \param[in] iVertex - Marker vertex index.
+   * \return Vertex normal heat fluxes (nVertex, nDim).
+   */
+  passivedouble GetMarkerNormalHeatFluxes(unsigned short iMarker, unsigned long iVertex) const;
+
+  /*!
+   * \brief Set the normal heat flux at a marker vertex.
+   * \param[in] iMarker - Marker index.
+   * \param[in] iVertex - Marker vertex index.
+   * \param[in] values - Vertex normal heat flux.
+   */
+  void SetMarkerNormalHeatFluxes(unsigned short iMarker, unsigned long iVertex, passivedouble values);
+
+  /*!
+   * \brief Get the number of nonequilibrium chemical species.
+   * \return Number of nonequilibrium chemical species.
+   */
+  unsigned long GetNumberNonequilibriumSpecies() const;
+
+  /*!
+   * \brief Get the number of nonequilibrium conservative state variables.
+   * \return Number of nonequilibrium conservative state variables.
+   */
+  unsigned long GetNumberNonequilibriumStateVariables() const;
+
+  /*!
+   * \brief Get the number of nonequilibrium primitive state variables.
+   * \return Number of nonequilibrium primitive state variables.
+   */
+  unsigned short GetNumberNonequilibriumPrimitiveVariables() const;
+
+  /*!
+   * \brief Get nonequilibrium chemical mass fractions.
+   * \param[in] iPoint - Point index.
+   * \return Nonequilibrium chemical mass fractions (nSpecies).
+   */
+  vector<passivedouble> GetNonequilibriumMassFractions(unsigned long iPoint) const;
+
+  /*!
+   * \brief Get nonequilibrium vibrational electronic temperatures.
+   * \return Nonequilibrium vibrational electronic temperatures (nPoint).
+   */
+  vector<passivedouble> GetVibrationalTemperatures() const;
```

Comment:
Is this not available via the access to primitive variables?

Reply:
Yeah, they should be accessible. Removed!